pkg/routeagent_driver/constants/constants.go

```diff
@@ -21,6 +21,10 @@ package constants
 const (
 	// IPTable chains used by RouteAgent
 	SmPostRoutingChain = "SUBMARINER-POSTROUTING"
+	PostRoutingChain   = "POSTROUTING"
+	MangleTable        = "mangle"
+	RemoteCIDRIPSet    = "remoteCIDRs"
+	LocalCIDRIPSet     = "localCIDRs"
```

Comment:
Just a nit, so it's more obvious when somebody inspect the system to who do those belong.
```suggestion
	RemoteCIDRIPSet    = "subm-remoteCIDRs"
	LocalCIDRIPSet     = "subm-localCIDRs"
```

Reply:
@aswinsuryan The max length for an IPset can be 31 chars, so we can use the full name itself.
i.e., SUBMARINER-REMOTECIDRS, SUBMARINER-LOCALCIDRS